lib/trento.pm

```diff
@@ -63,12 +68,95 @@ use constant TRENTO_AZ_PREFIX => 'openqa-trento';
 # Azure does not support dash or underscore in ACR name
 use constant TRENTO_AZ_ACR_PREFIX => 'openqatrentoacr';
 use constant CYPRESS_IMAGE_TAG => 'goofy';
+use constant CYPRESS_IMAGE => 'docker.io/cypress/included';
 
 =head1 DESCRIPTION 
 
 Package with common methods and default or constant  values for Trento tests
 
 =head2 Methods
+=cut
+
+=head3 get_trento_deployment
+
+Get the set of scripts for the Trento deployment
+=cut
+
+sub get_trento_deployment {
+    my ($self, $work_dir) = @_;
+
+    enter_cmd "cd $work_dir";
+    my $gitlab_token = get_var(TRENTO_GITLAB_TOKEN => get_required_var('_SECRET_TRENTO_GITLAB_TOKEN'));
+    script_run 'read -s GITLAB_TOKEN', 0;
+    type_password get_var(TRENTO_GITLAB_TOKEN => get_required_var('_SECRET_TRENTO_GITLAB_TOKEN')) . "\n";
```

Comment:
Secret management for the TRENTO_DEPLOY_VER part, L89 can be removed as figuring out how to use the env variable in the else

Reply:
Why not use `type_password $gitlab_token ...` instead of `type_password get_var(TRENTO_GITLAB_TOKEN => ...` in line 91?

Reply:
I like to try to remove line 89 at all in some next commits. Problem is how to manage the secret in the else from line 146

Reply:
Seems like in the block below L146 the only thing done with the token is to check whether it is the same as the one supplied. As such, one option would be to get rid of the check, and assume that the token stored in the qcow2 is the correct one. The other option would be to move the check to the shell, as you have the token already stored in a shell environment variable; for example you could check with:
```
assert_script_run '[[ "$GITLAB_TOKEN" == "$(git config --get remote.origin.url | cut -d@ -f1|cut -d: -f3)" ]]'
```
If they are the same, the above command will return 0 and pass, otherwise it will return 1 and `assert_script_run()` will fail the test.